macrosynergy/management/dq.py

```diff
@@ -236,8 +155,10 @@ def _fetch_threading(self, endpoint, params: dict):
 
         while clause(counter):
```

Comment:
Why are we defining a clause lambda function that hard-codes a max of 5 re-fetching?

Either it should be:
```python
while True:
   # TODO break point explicitly built into code
   pass
```
OR we should have it as a configurable state variable, if we have reason to believe there is a fixed set of responses:
```python
while counter <= 5:
    pass
```

Reply:
Got it, the counter is for the connection error/timeout. In that case I would skip the lambda function and write it directly as: `while counter <= 5` to make it clearer.

Reply:
Changed.

Included to try the four possible servers. If there is not a response after five tries, able to deduce that the servers are down.